ab_tests/ab_tests.yaml

```diff
@@ -13,3 +13,6 @@
 - SearchClusterQueryABTest:
   - A
   - B
+- LandingPageTest:
```

Comment:
Slightly weird that the naming convention here is inconsistent. Half the AB test names have a `ABtest` suffix and half don't. Not your fault though as there is both patterns before LandingPageTest even existed! 🙁

Reply:
Yeah - I went with the option without ABTest as the docs say the name should be prefixed with `ABTest-` but that this doesn't need to be included in the code as it's already configured in Fastly... the full name being `ABTest-LandingPageABTest` seemed a bit odd to me 🤷‍♀

Reply:
Yup I agree that the suffix is over kill. If I feel like a good Samaritan / brave enough I may make the naming consistent across all the 4 tests 😆